log/log.go

```diff
@@ -3,58 +3,60 @@ package log
 import (
 	"os"
 
-	"github.com/sirupsen/logrus"
+	"log"
+	"io"
 )
 
-// SetupLogging redirects logs to stderr and configures the log level.
-func SetupLogging(verbose bool) {
-	logrus.SetOutput(os.Stderr)
-	if verbose {
-		logrus.SetLevel(logrus.DebugLevel)
-	} else {
-		logrus.SetLevel(logrus.InfoLevel)
-	}
+type Logger interface {
+	Debug(format string, args ...interface{})
```

Comment:
Should we add the `-f` suffix, just to be consistent with the rest of go logging/printing functions? (Debugf, Infof, Errorf....)

Reply:
I also think so, but keep them without the `f` to maintain a similar previous api